https://github.com/pfig/net-amazon-s3/pull/42

From 116d73235d8243ec4427e6d7bf1f14f7fb9129aa Mon Sep 17 00:00:00 2001
From: Jay Hannah <jay.hannah@iinteractive.com>
Date: Thu, 8 May 2014 02:29:14 +0000
Subject: [PATCH] Patch for current versions of Moose: "Passing a list of
 values to enum is deprecated. Enum values should be wrapped in an arrayref."

---
 lib/Net/Amazon/S3/Client/Object.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/Net/Amazon/S3/Client/Object.pm b/lib/Net/Amazon/S3/Client/Object.pm
index 08e48c6..106f2fd 100755
--- a/lib/Net/Amazon/S3/Client/Object.pm
+++ b/lib/Net/Amazon/S3/Client/Object.pm
@@ -16,7 +16,7 @@ enum 'AclShort' =>
     [ qw(private public-read public-read-write authenticated-read) ];
 
 enum 'StorageClass' =>
-    qw(standard reduced_redundancy);
+    [ qw(standard reduced_redundancy) ];
 
 has 'client' =>
     ( is => 'ro', isa => 'Net::Amazon::S3::Client', required => 1 );
